requirements.unittest.txt

```diff
@@ -0,0 +1,17 @@
+#Install the agent
+-e .
+
+# app
+git+https://github.com/openstack/neutron@9.1.0
```

Comment:
@mattgreene noticed the same thing. This came from the old scripts, but agree that it needs to be changed to `stable/liberty`.